docker-compose.yml

```diff
@@ -21,9 +21,11 @@ services:
       FHIR_URI: "http://fhir-api:8099"
       ROLE_SERVICE_URI: "http://role-service:8082"
       SITE_SERVICE_URI: "http://site-service:8083"
+      PRACTITIONER_SERVICE_URI: "http://practitioner-service:8081"
```

Comment:
does the practitioner needs its own uri?

Reply:
The practitioner service has its own URI.  But following recent merges there are no changes to docker-compose in this PR.